### DISCUSSION TITLE
Repository is disabled for clone, push, pull

### DISCUSSION THREAD

**Discussion by purkayasta** [community/community#22454] [New to GitHub]
<p>I have changed the visibility of one of the projects from private to public and suddenly I can not push, pull to that repository. It is visible on the web but cannot push, pull, or even clone.</p>
<p>Whenever I tried to clone it got this error message<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/3/13cb0373d729fa61ddb9925314ad3bed9975ebf2.jpeg" data-download-href="/uploads/short-url/2P61DcsRjqzaLiC6hoXS20GWANQ.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/12936435/181071210-b7e7305b-1723-49e1-9093-6a66cd25cc57.jpeg" alt="image" data-base62-sha1="2P61DcsRjqzaLiC6hoXS20GWANQ" width="690" height="96" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/3/13cb0373d729fa61ddb9925314ad3bed9975ebf2_2_690x96.jpeg, https://aws1.discourse-cdn.com/github/original/3X/1/3/13cb0373d729fa61ddb9925314ad3bed9975ebf2.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/3/13cb0373d729fa61ddb9925314ad3bed9975ebf2.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/3/13cb0373d729fa61ddb9925314ad3bed9975ebf2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">912×127 26.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>I don’t understand what is the problem here or can anyone point me out what am I doing wrong and how to fix it?</p>
<p>this is the repo link: <a href="https://github.com/purkayasta/Fixerr" class="inline-onebox" rel="noopener nofollow ugc">GitHub - purkayasta/Fixerr: A minimal unofficial c# client for FIXER.IO</a></p>
<p>FYI: already opened a ticket still got no response from the team.</p>

**Comment by purkayasta** [comment]
<p>Update 1: If I change the visibility again from public to private it works. But why?</p>

**Comment by byrneh** [comment]
<p><a class="mention" href="/purkayasta">@purkayasta</a> strange its currently set to Public visibility and I was able to clone it just now.</p>
<blockquote>
<p>$ gh repo clone purkayasta/Fixerr<br>
Cloning into ‘Fixerr’…<br>
remote: Enumerating objects: 230, done.<br>
remote: Counting objects: 100% (230/230), done.<br>
remote: Compressing objects: 100% (129/129), done.<br>
remote: Total 230 (delta 118), reused 191 (delta 84), pack-reused 0<br>
Receiving objects:  76% (175/230)<br>
Receiving objects: 100% (230/230), 39.85 KiB | 658.00 KiB/s, done.<br>
Resolving deltas: 100% (118/118), done.</p>
</blockquote>

**Comment by purkayasta** [comment, marked as answer]
<p>yes, now it is possible.<br>
I have changed the visibility from public to private then again private to the public now it is working.</p>
<p>Thanks. :smiley:</p>
